city_local/london.md

```diff
@@ -43,6 +43,8 @@ attend any public TWC London meetings.
   slip through the cracks and not be granted access. If you have been waiting
   without a response for more than a week or two after signing up, please email
   or DM us on Twitter and we can try to help get you in.
+  
+- To learn more about TWC London and your rights as a tech worker in the U.K., you can read more in our zine, [Tech Workers Unite] (https://archive.org/stream/techworkersunite/TechWorkersUnite). 
```

Comment:
I think there's a space between the `]` and `(` here causing this not to be picked up as text for the link, at least according to the rich diff view on GitHub

Reply:
Thanks, Bob! Not totally sure I followed the right procedure here, but I've just made a new commit to address that error.

Reply:
Looks good, @shushugah can squash it when merging if needed 🙂